object/types.proto

```diff
@@ -204,3 +216,9 @@ message SplitInfo {
   // object parts.
   neo.fs.v2.refs.ObjectID link = 3;
 }
+
+// Lock keeps record of objects that are locked.
+message Lock {
```

Comment:
Agree, let's move it to a separate directory.

Reply:
May be we need to concentrate object content types in single package? Otherwise, for the sake of one type, we will create a new package/dir. Also it seems more

Reply:
There may be service calls added in the future, related to each type. New object types are rare, we don't expect them to grow in number in the nearest future, so let's have a new directory for locks.